db/seeds.rb

```diff
@@ -29,9 +31,36 @@
                city: city,
                state: state,
                zipcode: zipcode,
-               country: country )
+               country: country)
 end
 
+
+User.create!(name: 'Jeff Rust',
+             email: 'beazley2@pdx.edu',
```

Comment:
you probably wont want to leave your email in if this ends up being our final version